src/main/scala/org/broadinstitute/dsde/agora/server/dataaccess/permissions/PermissionsClient.scala

```diff
@@ -126,6 +126,25 @@ trait PermissionsClient {
 
   }
 
+  def listOwners(agoraEntity: AgoraEntity): Seq[String] = {
+    val permissionsQuery = for {
+      entity <- entities if entity.alias === alias(agoraEntity)
+      _permissions <- permissions if _permissions.entityID === entity.id // && _permissions.roles===Manage
```

Comment:
yeah, you should absolutely leverage the database to do this. Looking at other agora code such as `AgoraBusiness.filterEntityByRead()` ... please don't string together a list of `permission.roles === 1 || permission.roles === 3 || permission.roles === 5 ...`. Look at Slick's `inSetBind` instead.